### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by qsunyuan** [linghan1997/Regression-based-Analytic-Incremental-Learning#1]
Great work and thanks to the author for the open-source code. I have just started the continual learning task of VLM.

I would like to ask the author if the part of transfer acc starts directly from D2 (D1 and D2 in the upper-left corner of Fig. 2 in the original paper).

Also, the accuracy of zero-shot in my reproduction seems to be a little lower. Are there perhaps some other details? (I already set `eval_adapter: 0` in the yaml file.)
 https://github.com/linghan1997/Regression-based-Analytic-Incremental-Learning/blob/8ecfc7b6ab468bea2196ec2f7247ea4403c16c5f/configs/analytic_clip.yaml#L18

**Comment by linghan1997** [comment]
Thanks for your question!
--> _if the part of transfer acc starts directly from D2_
Yes. Please refer to the metric figure in the paper, where the blue block represents the transfer acc.

--> _the accuracy of zero-shot in my reproduction seems to be a little lower._
The zero-shot performance only depends on the pre-trained CLIP and the templates. Setting eval_adapter does not influence the performance, but only saves time for evaluation (avoid recording transfer acc repeatedly since it keeps consistent in our method).
